### ISSUE TITLE
Google Codelabs tool for learning material

### ISSUE THREAD

**Issue by dkastl** [pgRouting/admin#28]
Just found Google Codelabs, which provides nice looking tutorials: https://codelabs.developers.google.com/

So I was curious, if its source code is actually released as Open Source, which seems to be the case: https://github.com/googlecodelabs/tools

This could be an interesting alternative for howto's or a new workshop.

**Comment by cvvergara** [comment]
I tried one, 
https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html?index=..%2F..index#0
Uses
https://dartpad.dartlang.org/

I wonder if there is a web terminal for postgres somewhere

**Comment by dkastl** [comment]
Yes, some interactive console would be nice, but this probably requires some server instances, which usually cost money. Or there would be a simple way for users to setup this themselves.

But I liked the way the excercises are strutured with the progress on the left and information about renaming time. And it can be hosted independently, because the tool is Open Source and just generates a website. I would like to split the next generation of the workshop into a set of small excercises, that they can be done independently, if there is not enough time for the whole workshop.

**Comment by cvvergara** [comment]
What ever we use/decide for generating it, in general terms I like:

> I would like to split the next generation of the workshop into a set of small exercises, that they can be done independently, if there is not enough time for the whole workshop.

My comments on it:

But I feel that is more a "1 file per function" tutorial
or a "1 file per function per level" tutorial

For example
https://workshop.pgrouting.org/2.5/en/chapters/shortest_path.html

This is a page for 2 different pgRouting functions


To my mind comes the idea of using the sample data of the documentation for a tutorial section within the workshop. 
- A section that uses FOSS4G location data
- A section that use the sample data of the documentation (tutorial section)

This could help for 2 things:
1) moving [too many examples](https://docs.pgrouting.org/2.6/en/pgr_createVerticesTable.html#usage-when-the-edge-table-s-columns-match-the-default-values) to the tutorial section of the workshop  (my fault, I was learning SQL and [this tutorial](https://docs.pgrouting.org/2.0/en/doc/src/tutorial/index.html) was very difficult for me to understand and did not explained named parameters, so I made it for my understanding)
2) moving [advanced documentation](https://docs.pgrouting.org/2.6/en/dijkstra-family.html#the-problem-definition-advanced-documentation) to the "advanced tutorial" section of the workshop
(again my fault, because  the way it was handling the graph internally vs how the graph is stored in the database was not clear, so with math things become clear, again for my understanding)

Other way to do it is like what I did here:
http://talks.vicky.georepublic.info/howto/flipGeometries.html#/
